debian/control

```diff
@@ -63,6 +63,7 @@ Depends: binutils,
          sdparm | hdparm,
          uuid-runtime,
          xfsprogs,
+         cryptsetup,
```

Comment:
Maybe in Recommends instead of Depends ? And definitely in Build-Depends above so it's available at build time for make check to run.